full-service/src/service/txo.rs

```diff
@@ -339,6 +339,8 @@ where
                 subaddress_index: None,
             },
             None,
+            None, /* FIXME: I don't believe we need to provide spend_only_from_subaddress for
```

Comment:
Splitting TXOs is a very specific endpoint, and does not need to have `spend_only_from_subaddress` specified